Source/Core/InputCommon/ControlReference/ControlReference.cpp

```diff
@@ -48,6 +47,17 @@ ParseStatus ControlReference::GetParseStatus() const
   return m_parse_status;
 }
 
+std::string ControlReference::GetExpression() const
+{
+  return m_expression;
+}
+
+void ControlReference::SetExpression(std::string expr)
+{
+  m_expression = expr;
```

Comment:
Ah, I didn't know why the member functions weren't showing up in the output. So, what's going on in this case? Is it deciding at runtime whether to do a move or not?

> That's fair. Sorry about that.

Thanks, I appreciate that.